Makefile

```diff
@@ -5,12 +5,16 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 1.0.6
 
+ifneq (,$(shell which kubectl 2>/dev/null)$(shell which oc 2>/dev/null))
+	include build/make/deploy.mk
+endif
+
 # Add silent flag for all commands by default
 ifndef VERBOSE
 	MAKEFLAGS += --silent
 endif
 
-PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+PROJECT_DIR := $(shell dirname $(abspath $(firstword $(sort $(MAKEFILE_LIST)))))
```

Comment:
This seems to cause some problems with make bundle, I am investigating @tolusha

Reply:
I moreso meant getting rid of `$PROJECT_DIR` entirely in the Makefile, i.e. instead of `$PROJECT_DIR/some/path` it should be sufficient to do `./some/path`. I could be wrong though, I don't recall exactly the details here.